apple/internal/processor.bzl

```diff
@@ -702,7 +711,8 @@ def _process(
         process_and_sign_template,
         provisioning_profile = None,
         rule_descriptor,
-        rule_label):
+        rule_label,
+        env = None):
```

Comment:
[question] Why default value to None? Can this be mandatory?

Reply:
I did it because there may be other potential users for `_process` method that do not call it with `env` parameter. If we do it mandatory, it may break something in other peace of code that does not require env for itself.

Reply:
Could you test that, make `env` mandatory and see what other places are using this?
Context: 
* Not sure if there are any other places using this. Maybe you can make it mandatory and it just works
* In case of new code is written that needs this function, having this optional can lead to the same problem you are experiencing now